packages/openapi-typescript/package.json

```diff
@@ -38,7 +40,9 @@
     "url": "https://github.com/drwpow/openapi-typescript/issues"
   },
   "scripts": {
-    "build": "del dist && tsc -p tsconfig.build.json",
+    "build": "run-s -s build:*",
+    "build:esm": "tsc -p tsconfig.build.json",
+    "build:cjs": "esbuild --bundle --platform=node --target=es2019 --outfile=dist/index.cjs src/index.ts --footer:js=\"module.exports = module.exports.default;\"",
```

Comment:
With `dist/index.cjs`, the hope is that this will just shadow TypeScript types generated for `dist/index.js`. But we’ll see.

Reply:
Also the `--footer:js` is a hack to get rid of `require(…).default` without affecting the ESM package.

I spent some time trying to bundle with Rollup, but it turned out to be far more complex, and the end result still required similar hacks (just more in number)